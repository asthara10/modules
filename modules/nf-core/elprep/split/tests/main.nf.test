nextflow_process {

    name "Test Process ELPREP_SPLIT"
    script "../main.nf"
    config "./nextflow.config"
    process "ELPREP_SPLIT"

    tag "modules"
    tag "modules_nfcore"
    tag "elprep"
    tag "elprep/split"

    test("test_elprep_split") {


        when {
            process {
                """
                
                input[0] = [
        [ id:'test', single_end:false ], // meta map
        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true)
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
