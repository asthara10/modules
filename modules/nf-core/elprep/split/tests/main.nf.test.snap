{
    "versions": {
        "content": [
            [
                "versions.yml:md5,3b81f9795bdcba9513ad441ff5720938"
            ]
        ],
        "timestamp": "2023-11-24T10:31:44.745876"
    }
}